### ISSUE TITLE
continual red/green testing with a growl interface

### ISSUE THREAD

**Issue by rdpoor** [mjhea0/flaskr-tdd#43]
Lovely tutorial, and I'm a huge fan of TDD (and Flask, etc).  As icing on the cake, you might want to direct users to a red-light / green-light mechanism with continual testing and "growl" style icons.

I've written a system like that for macOS, but I'm sure there are equivalent packages for Windows and Linux.  

(And I don't suggest you bloat your tutorial with this kind of thing but rather provide a link.)

Anyway, great stuff.

**Comment by mjhea0** [comment]
Is the system that you have open source?

**Comment by rdpoor** [comment]
It is now!  I just added an MIT license and pushed it to:

   https://github.com/rdpoor/auto-red-green

As it's only a tool I wrote for my own use, there no ReadMe yet, but it's not that complicated.  You'll need to customize the run-tests script for your own setup.  A future version should take command line args, etc.  

Anyway, I'd welcome suggestions, critiques, pull requests, etc.

(And I'm 95% through flaskr-tdd.  It's really pleasant.  Thanks again.)
